src/lib/serviceRoutines/statisticsTreat.cpp

```diff
@@ -394,101 +212,165 @@ std::string statisticsTreat
   // Instead of removing version-requests from the statistics,
   // we report the number of version-requests even if zero (-1).
   //
-  out += TAG_ADD_COUNTER("versionRequests", noOfVersionRequests);
+  js.addNumber("versionRequests", noOfVersionRequests + 1);
+
+  renderUsedCounter(&js, "exitRequests", noOfExitRequests);
+  renderUsedCounter(&js, "leakRequests", noOfLeakRequests);
+  renderUsedCounter(&js, "statisticsRequests", noOfStatisticsRequests);
+  renderUsedCounter(&js, "invalidRequests", noOfInvalidRequests);
+  renderUsedCounter(&js, "registerResponses", noOfRegisterResponses);
+  renderUsedCounter(&js, "registrationErrors", noOfRegistrationErrors);
+  renderUsedCounter(&js, "registrationUpdateErrors", noOfRegistrationUpdateErrors);
+  renderUsedCounter(&js, "discoveryErrors", noOfDiscoveryErrors);
+
+  // FIXME P10: old implementation overriden by mscRefreshs, mscInserts, mscRemoves, mscUpdates, cacheItems
```

Comment:
Yes, this seems to come from the initial implementation of the sub-cache.
Can be removed.